flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorTest.java

```diff
@@ -1459,7 +1420,7 @@ public void testReconnectionAttemptIfExplicitlyDisconnected() throws Exception {
 	 */
 	@Test
 	public void testInitialSlotReport() throws Exception {
-		final TaskSlotTable taskSlotTable = new TaskSlotTable(Collections.singleton(ResourceProfile.ANY), timerService);
+		final TaskSlotTable taskSlotTable = TaskSlotUtils.createTaskSlotTable(1);
```

Comment:
why are you no longer passing the timerService? (also applies to many other test methods)

Reply:
they all use the default timer service, same as in the builder by default, which is closed when TM terminates and slot table is stopped. I will add a hotfix commit to remove this test class field `timerService` as it is redundant after this refactoring.